hls/xls/meta.yaml

```diff
@@ -0,0 +1,53 @@
+# Use `conda-build-prepare` before building for a better version string.
+{% set version = '%s_%04i_%s'| format(GIT_DESCRIBE_TAG | default('0.0.0') , GIT_DESCRIBE_NUMBER | default(0) | int, GIT_DESCRIBE_HASH | default('gUNKNOWN')) %}
+
+package:
+  name: xls
+  version: {{ version }}
+
+source:
+  - git_url: https://github.com/google/xls.git
+    git_rev: main
+    patches:
+      - add-dist-rule.patch
+      - skip-python-requirements.patch
+
+build:
+  # number: 202202082028
+  number: {{ environ.get('DATE_NUM') }}
+  # string: 20220208_2028
+  string: {{ environ.get('DATE_STR') }}
+  script_env:
+    - CI
+
+requirements:
+  build:
+    - bazel
+  host:
+    - python
+    - pip
+    - pycparser
+    - Flask
+    - Jinja2
+    - werkzeug
+    - itsdangerous
+    - click
+    - protobuf
```

Comment:
Is it correct? It's absent in the https://github.com/google/xls/blob/692e37534dfcd1af41b0b8b04ff2004d6a8193c5/dependency_support/pip_requirements.txt

Reply:
I think that was needed because of https://github.com/google/xls/blob/629d348b6320d912206701afe064f28fa85ca948/xls/solvers/python/BUILD#L63